Godeps/Godeps.json

```diff
@@ -1682,6 +1682,46 @@
 		{
 			"ImportPath": "gopkg.in/yaml.v2",
 			"Rev": "7ad95dd0798a40da1ccdff6dff35fd177b5edf40"
+		},
```

Comment:
No, I don't think so. They were added by godeps but it looks to be a mistake. I removed these files but forgot to update Godeps.json. I'll drop them.